lib/src/stack/coap_chain.dart

```diff
@@ -421,7 +421,7 @@ class CoapChain<TChain, TFilter, TNextFilter>
   }
 
   CoapEntry<dynamic, dynamic> _checkOldName(String baseName) =>
-      _name2entry[baseName] as CoapEntry<dynamic, dynamic>;
+      _name2entry[baseName]! as CoapEntry<dynamic, dynamic>;
```

Comment:
I think this null check operator is also not needed.

Reply:
I think in general it's better to be explicit in the code. In this case the cost is also only one character to clarify that the variable can be null, but that we are not expecting it to be?

Reply:
Yeah, also considering your last comment, it is certainly better to leave it as for now :)

Reply:
Ok! Marking this as resolved now then :)